website/blog/2020-07-22-62updates.md

```diff
@@ -0,0 +1,48 @@
+
+![fast-refresh-on-windows](assets/fastrefresh.gif)
+
+## Microsoft.ReactNative APIs
+Several updates have been made to provide thread safe, higher performance instance management on React Native for Windows through new Microsoft.ReactNative APIs. The deprecated ReactUwp framework is no longer supported with 0.62. Read our documentation on these for more, few highlights:
+* [ReactApplication](https://microsoft.github.io/react-native-windows/docs/reactapplication-api) : Base Application class for use in applications that are entirely written in react-native
+* [ReactNativeHost](https://microsoft.github.io/react-native-windows/docs/reactnativehost-api) : Entry point to create a new react-native instance
+* [ReactInstanceSettings](https://microsoft.github.io/react-native-windows/docs/reactinstancesettings-api) : Provides configuration of the react-native instance
+* [ReactContext](https://microsoft.github.io/react-native-windows/docs/ireactcontext-api) : Given to native modules to communicate with other native modules, views, application, and the react-native instance
+* [ReactPropertyBag](https://microsoft.github.io/react-native-windows/docs/ireactpropertybag-api) : Provides a thread-safe property storage
+* [XamlUIService](https://microsoft.github.io/react-native-windows/docs/xamluiservice-api) : Provides access to XAML UI specific functionality. See [Native UI components](https://microsoft.github.io/react-native-windows/docs/view-managers) for more details on the changes to `ViewManager` APIs.
+* [RedBoxHandler](https://microsoft.github.io/react-native-windows/docs/iredboxhandler-api) : Provides an extension point to allow custom error handling within the react-native instance
+
+## Alignment with core
+As part of our ongoing alignment with React Native core we have added Windows support for several new APIs introduced in ReactNative 0.62. Few callouts here:
+   * [Appearance](https://reactnative.dev/docs/appearance) module is now supported on Windows. You can use this module to get information on the user's dark/light theme preferences and apply the right styling as needed. Windows previously had `AppTheme` APIs that are now deprecated in favor of aligning with React Native core. 
+   * [focusable](https://reactnative.dev/docs/view#focusable) prop is now supported on `View` and `TextInput`. You can use this prop to indicate whether a component can be focusable (participate in tab order) when navigating with keyboard. Windows previously had an `acceptsKeyboardFocus` prop that is now deprecated in favor of aligning with React Native core. 
+   * **RTL Support in Windows** : We have added support to detect when an app is running in RTL locale and automatically apply the RTL [LayoutDirection](https://reactnative.dev/docs/flexbox#layout-direction). In addition, Windows now supports `forceRTL` and `allowRTL` modules.
```

Comment:
>* **RTL Support in Windows** : We have added support to detect when an app is running in RTL locale and automatically apply the RTL [LayoutDirection](https://reactnative.dev/docs/flexbox#layout-direction). In addition, Windows now supports `forceRTL` and `allowRTL` modules. [](start = 3, length = 274)

We have this issue outstanding:  [https://github.com/microsoft/react-native-windows/issues/4770](https://github.com/microsoft/react-native-windows/issues/4770).  This means that the app won't automatically flip to RTL in RTL locales.  Probably best to leave this off the list for this release.

Reply:
Thanks Keith, I updated the text to reflect this.